local.go

```diff
@@ -193,6 +193,11 @@ func (l *localImage) SetLabel(key, val string) error {
 	if l.inspect.Config == nil {
 		return fmt.Errorf("failed to set label, image '%s' does not exist", l.repoName)
 	}
+
+	if l.inspect.Config.Labels == nil {
+		l.inspect.Config.Labels = make(map[string]string)
```

Comment:
minor stylistic point: our convention in this codebase has been to use `map[string]string{}` instead of `make(map[string]string)`